src/contexts/globalLayout/GlobalNavbar/LoggedInNav.tsx

```diff
@@ -95,10 +93,10 @@ function LoggedInNav({ queryRef }: Props) {
       <NavElement>
         <StyledDropdownWrapper hasNotification={false}>
           <Dropdown mainText={username || 'ACCOUNT'} shouldCloseOnMenuItemClick>
-            <TextButton
-              text="My Gallery"
-              onClick={username ? () => push(`/${username}`) : undefined}
-            />
+            <UnstyledLink href={`/${username}`}>
+              <TextButton text="My Gallery" />
+            </UnstyledLink>
+            <Spacer height={12} />
```

Comment:
not sure where this spacer came from but it adds an unwanted space in the dropdown. i'll toss it!